website/content/docs/integrations/vault/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+page_title: Vault Integration
+description: |-
+ How to integrate Vault and Boundary
+---
+
+# Boundary and Vault overview
```

Comment:
```suggestion
# Boundary and Vault integrations
```

Reply:
There might be a better title for this. I don't think "overview" is very descriptive here, but my suggestion "integrations" isn't either. Since this topic lays out the potential ways to configure a Boundary + Vault integration, maybe "Boundary and Vault integration options" would be better, or "Boundary and Vault integration potential". What do you think?

I don't think we want to call it "How to integrate Boundary and Vault" because users will come here for step-by-step instructions, and this topic is more conceptual.

Reply:
Agreed! A "How to...." implies a step by step guide. I think "Boundary and Vault Integration Options" is the better out of the two. I have plans to include a section on Terraform within this integrations section too

Reply:
> Agreed! A "How to...." implies a step by step guide. I think "Boundary and Vault Integration Options" is the better out of the two. I have plans to include a section on Terraform within this integrations section too

Sounds good to me! 👍